platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -0,0 +1,1027 @@
+</details>
+<details>
+  <summary>**Organizations**</summary>
+
+  Run `tw organizations -h` to view supported workspace operations.
+
+  Organizations are the top-level structure and contain workspaces, members, and teams. You can also add external collaborators to an organization. See [Organization management](../orgs-and-teams/organizations.mdx) for more information. 
+
+  ### Add an organization
+
+  Run `tw organizations add -h` to view the required and optional fields for adding your workspace.
+
+  ```console 
+  tw organizations add -n TestOrg2 -f 2nd\ Test\ Organization\ LLC -l RSA
+
+  Organization 'TestOrg2' with ID '204336622618177' was added
+  ```
+
+</details>  
+<details>
+  <summary>**Members**</summary>
+
+  Run `tw members -h` to view supported member operations.
+
+  Manage organization members. 
+
+  ### List members 
+
+  Run `tw members list -h` view all the optional fields for listing organization members.
+
+  ```console
+  tw members list -o TestOrg2
+
+  Members for TestOrg2 organization:
+
+     ID              | Username             | Email                           | Role
+    -----------------+----------------------+---------------------------------+--------
+     200954501314303 | user1                | user1@domain.com                | MEMBER
+     277776534946151 | user2                | user2@domain.com                | MEMBER
+     243277166855716 | user3                | user3@domain.com                | OWNER
+
+  ``` 
+
+  ### Add a member 
+
+  Run `tw members add -h` view all the required and optional fields for adding organization members.
+
+  ```console
+  tw members add -u user1@domain.com -o DocTestOrg2
+
+  Member 'user1' with ID '134534064600266' was added in organization 'TestOrg2'
+  ```
+
+  ### Delete a member
+
+  ```console
+  tw members delete -u user1 -o TestOrg2
+
+  Member 'user1' deleted from organization 'TestOrg2'
+  ```
+
+  ### Update member role 
+
+  ```console 
+  tw members update -u user1 -r OWNER -o TestOrg2
+
+  Member 'user1' updated to role 'owner' in organization 'TestOrg2'
+  ```
+
+  ### Leave an organization
+
+  Run `tw members leave -o <organization_name>` to be removed from the given organization's members.
+
+</details>  
+<details>
+  <summary>**Workspaces**</summary>
+
+  Run `tw workspaces -h` to view supported workspace operations.
+
+  Workspaces provide the context in which a user launches workflow executions, defines the available resources, and manages who can access those resources. Workspaces contain pipelines, runs, actions, datasets, compute environments, credentials, and secrets. Access permissions are controlled with participants, collaborators, and teams.
+
+  See [User workspaces][user-workspaces] for more information.
+
+  :::note
+  This section assumes that you already have access to an organization in Seqera Platform.
+  :::
+
+  ### Add a workspace
+
+  Run `tw workspaces add -h` to view the required and optional fields for adding your workspace.
```

Comment:
Does this require a level of permissions to execute this command (like organization `ADMIN` role and above)?